ApexCollectionType.cls

```diff
@@ -0,0 +1,75 @@
+public class ApexCollectionType {
+    
+    public static void practiceOnMap(){
+        Map<String, List<String>> mapToStoreDepartmentAndRelatedEmployee = new Map<String, List<String>>();
+        
```

Comment:
Remove unwanted spaces.

Reply:
Sir, can I add one line space to differentiate the operations(Union, intersection etc.) and to enhance the readability or remove them also?

Reply:
Do whatever you want, but the code and the class should be properly indented.